### PR TITLE
Formatting

### PR DESCRIPTION
**Pull request by Leonievb**
Improved and adjusted formatting and documentation:
- all files are now in TSV file format
- the documentation on the usage of the three positional arguments has been improved
- DNA sequences representing a `clone_id` are consistently called clone_id, while numbers used to replace them were renamed to `clone#`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -263,8 +290,8 @@ A table listing the cell IDs belonging to each clone.
 The columns are *clone_id* and *cell_id* where *clone_id* is a number that
 identifies the clone.
 
-    #clone_id,cell_id
-    1,TGGCGCAAGAATAGGG
+    clone#   cell_id
+    1   TGGCGCAAGAATAGGG
```
Comment (marcelm) [inline README.md:294]:
```suggestion
    clone#   cell_id
    1        TGGCGCAAGAATAGGG
```

File: README.md [hunk 2]
```diff
@@ -273,8 +300,8 @@ A table listing the 30N sequence of each cloneID.
 The columns are *clone_id* and *clone_seq* where *clone_id* is a number
 that identifies the clone and *clone_seq* its nucleotide sequence.
 
-    #clone_id,clone_seq
-    1,ACTAGGAGATTGACGGATCACCTTTGGTCG
+    clone#   clone_seq
+    1   ACTAGGAGATTGACGGATCACCTTTGGTCG
```
Comment (marcelm) [inline README.md:304]:
```suggestion
    clone#  clone_seq
    1       ACTAGGAGATTGACGGATCACCTTTGGTCG
```

File: README.md [hunk 3]
```diff
@@ -273,8 +300,8 @@ A table listing the 30N sequence of each cloneID.
 The columns are *clone_id* and *clone_seq* where *clone_id* is a number
```
Comment (marcelm) [inline README.md:300]:
```suggestion
The columns are *clone#* and *clone_seq* where *clone_id* is a number
```

File: README.md [hunk 4]
```diff
@@ -160,10 +161,28 @@ This is an overview of the steps that the `trex run10x` command performs.
...
+
+### Filter cellids
+
+Tab-separated file of cell IDs to keep in the trex run. Adding this file via 
+the `--filter-cellid` or `-f` allows to focus the analysis on specific cells
+and to filter out low quality cells or doublets.
+Example:
+	x
+0	CACTCGTGGTACACACTCCG
+1	CACTCGTGGTACCACAAGCA
```
[the remark this answers is not in the document]
Comment (Leonievb) [reply README.md:180]: This comment confuses me. Why triple backticks before and after here but not before and after the other format examples?

File: README.md [hunk 5]
```diff
@@ -160,10 +161,28 @@ This is an overview of the steps that the `trex run10x` command performs.
...
+## Input files
+
+
+### Filter cellids
+
+Tab-separated file of cell IDs to keep in the trex run. Adding this file via 
+the `--filter-cellid` or `-f` allows to focus the analysis on specific cells
+and to filter out low quality cells or doublets.
+Example:
+	x
```
[the remark this answers is not in the document]
Comment (Leonievb) [reply README.md:178]: Solved. ```x``` was removed and ```header=None``` added

File: README.md [hunk 6]
```diff
@@ -160,10 +161,28 @@ This is an overview of the steps that the `trex run10x` command performs.
...
+- A useable read does not require a UMI
+- Reads do not get grouped into molecules and their cloneIDs not collapsed 
+  and error-corrected into consensus sequences
+
+## Input files
+
+
+### Filter cellids
+
+Tab-separated file of cell IDs to keep in the trex run. Adding this file via
```
[the remark this answers is not in the document]
Comment (Leonievb) [reply README.md:174]: Solved.

File: README.md [hunk 7]
```diff
@@ -177,7 +196,7 @@ or in comma-separated values (CSV) format.
 
 ### `log.txt`
 
-This file contains a copy of the output that `trex run10x` prints to the
+This file contains a copy of the output that a trex run prints to the
```
[the remark this answers is not in the document]
Comment (Leonievb) [reply README.md:199]: Solved.

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Leonievb) [reply README.md:180]: Solved.

**Comment by marcelm** [comment]
Looks good!

**Pull request merged**